app/src/release/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,20 @@
+<resources>
+    <!--
+    TODO: Before you release your application, you need a Google Maps API key.
+
+    To do this, you can either add your release key credentials to your existing
+    key, or create a new key.
+
+    Note that this file specifies the API key for the release build target.
+    If you have previously set up a key for the debug target with the debug signing certificate,
+    you will also need to set up a key for your release certificate.
+
+    Follow the directions here:
+
+    https://developers.google.com/maps/documentation/android/signup
```

Comment:
Is this something we need to do for the debug/test versions too?

Reply:
Yes, I removed the actual key from the commit, but we'll have to create a new api key for each build version.